c/index.html

```diff
@@ -14,12 +14,8 @@
             <a class="nav-link" href="#startc">Get started</a>
             <ul class="nav">
               <li class="nav-item"><a class="nav-link" href="#creq">Requirements</a></li>
-              <li class="nav-item"><a class="nav-link" href="#cinstall">Installation</a></li>
-              <li class="nav-item"><a class="nav-link" href="#cinstallosx">Installation on Mac OS X</a></li>
```

Comment:
The "Installation on Mac OS X" heading was removed - is that intentional? I'd keep it and simply rename "Mac OS X" to "macOS"